src/did/resolver.ts

```diff
@@ -0,0 +1,27 @@
+import { Resolver, DIDDocument } from "did-resolver";
+import { getResolver as ethrGetResolver } from "ethr-did-resolver";
+import { getResolver as webGetResolver } from "web-did-resolver";
+import { INFURA_API_KEY } from "../config";
+
+const providerConfig = {
+  networks: [{ name: "mainnet", rpcUrl: `https://mainnet.infura.io/v3/${INFURA_API_KEY}` }],
+};
+
+export const resolver = new Resolver({
+  ...ethrGetResolver(providerConfig),
+  ...webGetResolver(),
+});
+
+// TODO replace cache with timeout
```

Comment:
what do you mean ? add a max-age for expiry ?

Reply:
Yes, I'm not trying to get everything production worthy now

Reply:
Replaced with node-cache with 5 min TTL